src/pages/suunnitelma/[oid]/suunnittelu.tsx

```diff
@@ -209,7 +209,14 @@ const VuorovaikutusTiedot: FunctionComponent<{
       </ContentSpacer>
       <ContentSpacer>
         <H4>{t("tilaisuudet.tulevat_tilaisuudet")}</H4>
-        {tulevatTilaisuudet?.length ? <TilaisuusLista tilaisuudet={tulevatTilaisuudet} /> : <p>{t("tilaisuudet.julkaistaan_pian")}</p>}
+        {tulevatTilaisuudet?.length ? (
+          <TilaisuusLista tilaisuudet={tulevatTilaisuudet} />
+        ) : (
+          <p>
+            {t("tilaisuudet.kiitos_osallistumisesta")}{" "}
+            {isProjektiStatusGreaterOrEqualTo(projekti, Status.NAHTAVILLAOLO) ? t("tilaisuudet.kaikki_vastaanotetut") : null}
```

Comment:
eikö tässä tekstien pitäisi olla toistepäin, eli "kaikki vastaanotetut..." ei tule enää suunnitteluvaiheen jälkeen

Reply:
Hyvä havainto, ajattelin jotenkin ristiin.